src/main/java/com/rtb/projectmanagementtool/user/UserData.java

```diff
@@ -19,21 +19,21 @@
   private long userID;
   private long AuthID; // this will be the ID from API
   private String userName;
-  private int userYear;
-  private HashSet<String> userMajors;
+  private long userYear;
+  private ArrayList<String> userMajors;
   // private Skills skills;
   // private String skillsString;
-  private int userTotalCompTasks;
+  private long userTotalCompTasks;
```

Comment:
total number of completed tasks?  This could be ok, but keep in mind when they complete a task you'd now then have to update both the task object and this user object.